cmd/validator/validate_accounting.go

```diff
@@ -307,7 +320,9 @@ func resultsPrinter(config Params, printCurl bool, results <-chan Result) {
 
 			// Print error message if there is one.
 			if r.Error != nil {
+				numErrors++
 				errorLog.Printf("Processor error: %v\n", r.Error)
+				continue
 			}
 			// Print error details if there are any.
```

Comment:
So if r.Error == nil, !r.Equal but r.Details == nil, we don't need to increment numErrors?

Reply:
I'll make the error increment unconditional here. There should always be details but no need to depend on it.